CONTRIBUTING.md

```diff
@@ -155,21 +163,21 @@ $ pdm setup_env
 Becomes:
 
 ```sh
-$ pdm run nox -s docs
-$ pdm run nox -s lint
-$ pdm run nox -s pyright
-$ pdm run nox -s test
-$ pdm run noxfile.py -s dev
+$ uv run nox -s docs
+$ uv run nox -s lint
+$ uv run nox -s pyright
+$ uv run nox -s test
+$ uv run noxfile.py -s dev
 ```
 
-`nox` may also be installed to your path using either [uv](https://docs.astral.sh/uv) or [pipx](https://pipx.pypa.io) and used on the root: eg `nox -s docs` with no preceding `pdm run`
+`nox` may also be installed to your path using either [uv](https://docs.astral.sh/uv) or [pipx](https://pipx.pypa.io) and used on the root: eg `nox -s docs` with no preceding `uv run`
 
 ```sh
 # without global nox
-$ pdm run noxfile.py -s dev
+$ uv run noxfile.py -s dev
 
 # with global nox
```

Comment:
side note: nox doesn't have to be installed globally for this, one could also just activate the venv

Reply:
Correct.

Reply:
Now I see what you're commenting on.

Because the dev session exists to bootstrap the environment, the command provided is one that will work even without that environment present. Such that a user can clone the repository, and simply run `uv run noxfile.py -s dev` to bootstrap the local environment, its intentionally *not* using the local folder, because it sets it up.